docs/tutorial/data/index.rst

```diff
@@ -47,11 +61,15 @@ PCDPointCloud
 
 Colored point cloud of a living room from the Redwood dataset in PCD format.
 
+**Python:**
```

Comment:
`**Python:**`, and `**C++**` Take up a lot of space. Also, it is already clear by syntax. Please revert changes.
In future, we may shift to code tabs if required. 
Same for the rest of this file.

Reply:
Duly noted. If you **can** use code tabs, please do so (even though they [tabs] might also take up some screen real estate, but they would definitely look a lot more attractive).